modules/process-testing/pages/task.adoc

```diff
@@ -1,3 +1,106 @@
-= Tasks
+= Tasks in integration tests
+:description: Manage tasks in integration tests using the Bonita test toolkit
 
-_Coming soon_
\ No newline at end of file
+Learn to execute tasks and to validate their states in integration tests using the Bonita test toolkit.
+
+[NOTE]
+====
+Integrations tests are built upon the Bonita Test Toolkit, based on the open source https://github.com/bonitasoft/bonita-java-client[Bonita Java Client]. +
+The Bonita Test Toolkit is only available for Enterprise, Performance, Efficiency and Teamwork. editions. 
+====
+
+== Retrieve tasks
+
+Any task (user task, service task...) defined on a process can be retrieved through the xref:process.adoc#_tasks[process instance]. +
+Keep in mind that only tasks that *have been executed or are ready* can be retrieved, incoming tasks are not accessible.
+
+A task is returned as an object `Task` or `UserTask` (which extends `Task`). + 
+It offers the possiblity to retrieve sone meta information about the task, its state, and gives access to eventual xref:variable.adoc[variables] and xref:connector.adoc[connectors].
+
+Besides that, a user task provides information about candidates, assignee, due date, and can be assigned and executed.
+
+== Assign and execute user tasks
+
+User tasks can be executed from using the corresponding process instances to advance processes: 
+
+[source, Java]
+----
+@BonitaTests
+class ProcessIT {
+
+    @Test
+    void assign_and_execute_tasks(BonitaTestToolkit toolkit) {
+        User walter = toolkit.getUser("walter.bates");
+        ProcessDefinition processDef = toolkit.getProcessDefinition("myProcess");
+        ProcessInstance processInstance = processDef.startProcessFor(walter);
+        Task userTask = processInstance.getFirstPendingUserTask("aTask"); <1>
+        Contract contract = createContract(); <2>
+
+        // Check the candidates of a task
+        Set<User> candidates = userTask.getCandidates(); <3>
+        assertThat(candidates).extracting(User::getUserName).containsExactly(walter.getUserName());
```

Comment:
As equal is properly implemented, you can directly write

```java
 assertThat(candidates).containsExactly(walter);
```
We also have a `hasCandidates` predicate in case the actor filter is long to execute.